test/Dapr.Client.Test/StateApiTest.cs

```diff
@@ -224,6 +224,135 @@ public async Task SetStateAsync_ThrowsForNonSuccess()
             await FluentActions.Awaiting(async () => await task).Should().ThrowAsync<RpcException>();
         }
 
+        [Fact]
+        public async Task ExecuteStateTransactionAsync_CanSaveState()
+        {
+            // Configure Client
+            var httpClient = new TestHttpClient();
+            var daprClient = new DaprClientBuilder()
+                .UseGrpcChannelOptions(new GrpcChannelOptions { HttpClient = httpClient })
+                .Build();
+
+            var stateValue1 = new Widget() { Size = "small", Color = "yellow", };
+            var metadata1 = new Dictionary<string, string>()
+            {
+                {"a", "b" }
+            };
+            var options1 = new StateOptions();
+            options1.Concurrency = ConcurrencyMode.LastWrite;
+            ;
+            var state1 = new StateTransactionRequest("stateKey1", JsonSerializer.SerializeToUtf8Bytes(stateValue1), StateOperationType.Upsert, "testEtag", metadata1, options1);
+
+            int stateValue2 = 100;
```

Comment:
nitpick: try to stick to a style. So far I think everything in the repo uses `var`.